scene/gui/item_list.cpp

```diff
@@ -1185,6 +1181,12 @@ void ItemList::_notification(int p_what) {
 
 					Point2 pos = items[i].rect_cache.position + icon_ofs + base_ofs;
 
+					if (icon_mode == ICON_MODE_TOP) {
+						pos.y += theme_cache.v_separation / 2;
```

Comment:
Yes, it is intended. It ensures that elements are drawn pixel-perfect. Decimal values can make things blurry, which I want to avoid.

It is not necessary to convert the result to integer because both operands are ints already.